Samples/RadarIO.Xamarin.iOS.Sample/AppDelegate.cs

```diff
@@ -17,7 +17,7 @@ public class AppDelegate : UIApplicationDelegate, IUIApplicationDelegate
 
         public override bool FinishedLaunching(UIApplication application, NSDictionary launchOptions)
         {
-            Demo.Initialize();
+            Demo.Initialize(new RadarSDKImpl());
```

Comment:
So I guess now users will have to create this impl when initializing the SDK?  Can you help me understand why the need for this change?  

We would have to change this documentation, yes? https://radar.com/documentation/sdk/xamarin

Reply:
Good catch. Yes, unfortunately there is no way around this, I spent a lot of time searching for one. It wasn't needed before because I was doing a sort of compilation hack where I would swap in a platform-targeted binary with the same name as the .net standard library (i.e. iOS or Android) but it had the platform specific implementation in it so it could be resolved automatically (without passing in this new parameter). Unfortunately this was causing the build to randomly break and after some visual studio patch in the past year it would break the build 100% percent of the time. According to the Microsoft docs the correct way to use platform specific implementations in standard code is to use dependency injection (as this change is doing), which makes me think what I had working before was really just a hack that will never work again.

I can dig into it again if you'd like, but I spent quite a while on it already with no luck.

Yes, the documentation will need to be changed, specifically a call to `RadarSingleton.Initialize(new RadarSDKImpl())` will need to be added before using the Radar singleton (`Radar.Initialize(publishableKey);`)